juriscraper/opinions/united_states/federal_bankruptcy/bap1.py

```diff
@@ -0,0 +1,100 @@
+"""
+Scraper for the United States Bankruptcy Appellate Panel for the First Circuit
+CourtID: bap1
+Court Short Name: 1st Cir. BAP
+"""
+from datetime import datetime, timedelta
+from typing import Dict
+
+from lxml.html import HtmlElement
+
+from juriscraper.AbstractSite import logger
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.url = "https://www.bap1.uscourts.gov/bapopn"
+        self.court_id = self.__module__
+
+        self.method = "GET"
+
+        # Search the last month
+        date_filter_end = datetime.today()
+        date_filter_start = date_filter_end - timedelta(30)
+        self.parameters = {
+            "params": {
+                "opn": "",
+                "field_opn_short_title_value": "",
+                "field_opn_issdate_value[min][date]": date_filter_start.strftime(
+                    "%m/%d/%Y"
+                ),
+                "field_opn_issdate_value[max][date]": date_filter_end.strftime(
+                    "%m/%d/%Y"
+                ),
+            }
+        }
+
+        self.back_scrape_iterable = ["placeholder"]
```

Comment:
its not going to be a placeholder in this instance.  And you may be tripped up by me - make the iterable something else - like a list from 1 to 29 which would represent each page we want to grab from